deps.edn

```diff
@@ -18,7 +18,12 @@
         hiccup/hiccup {:mvn/version "2.0.0-alpha2"}
         lambdaisland/uri {:mvn/version "1.11.86"}
 
-        rewrite-clj/rewrite-clj {:mvn/version "1.0.699-alpha"}}
+        rewrite-clj/rewrite-clj {:mvn/version "1.0.699-alpha"}
+
+        ;; needed for CSS jit pass
+        babashka/process {:mvn/version "0.0.2"}
+        io.github.nextjournal/cas {:git/url "git@github.com:nextjournal/cas"
```

Comment:
Damn, this is a private repo :-(

Reply:
Don’t understand why we would need that. Think we should just have the css locally, and uploading it should be an independent concern, same as we don’t upload the html files.